frontend/packages/console-shared/locales/en/console-shared.json

```diff
@@ -289,6 +289,8 @@
   "Provides a list of all the available sub-catalog types which are shown in the Developer Catalog. The types must be added below spec customization developerCatalog": "Provides a list of all the available sub-catalog types which are shown in the Developer Catalog. The types must be added below spec customization developerCatalog",
   "Add user perspectives": "Add user perspectives",
   "Provides a list of all the available user perspectives which are shown in the perspective dropdown. The perspectives must be added below spec customization.": "Provides a list of all the available user perspectives which are shown in the perspective dropdown. The perspectives must be added below spec customization.",
+  "Add pinned resources": "Add pinned resources",
+  "Provides a list of all the available resources that can be pinned on the Developer perspective navigation. The pinned resources must be added below spec customization perspectives.": "Provides a list of all the available resources that can be pinned on the Developer perspective navigation. The pinned resources must be added below spec customization perspectives.",
```

Comment:
Since we are adding only the Samples but not all available resources, can we change the description? Description says, "Provides a list of **all the available resources**"